ui/src/app/pages/profile/data-use-agreement-panel.tsx

```diff
@@ -0,0 +1,139 @@
+  </div>;
+};
+
+export const SanctionModal = (props) => {
+  return <Modal width={750}>
+    <ModalTitle style={dataUseAgreementStyles.sanctionModalTitle}>
+      All of Us Research Program - Sanctions on Violations of the Code of Conduct
+    </ModalTitle>
+    <ModalBody>
+      <label style={dataUseAgreementStyles.modalLabel}>
+        The Resource Access Board (RAB) of the <i>All of Us</i> Research Program determines whether
+        an investigator has violated the Code of Conduct outlined in the Data Use Agreement
+        signed by each user. The RAB notifies the All of Us Research Program office of the
+        violation.
+        <IndentedUnorderedList>
+          <li>
+            <label>The <i>All of Us</i> Research Program office and/or the All of Us IRB may
+              implement the following sanctions if it is determined that an investigator has violated
+              the Code of Conduct:
+            </label>
+            <IndentedUnorderedList>
+              <IndentedListItem>
+                Determine whether any action by the investigator is required to remedy the
+                violation.
+              </IndentedListItem>
+              <IndentedListItem>
+                Revoke and/or deny access of the violator to all non-public (Registered and
+                Controlled tier) <i>All of Us</i> data.
+              </IndentedListItem>
+              <IndentedListItem>
+                Post the name and affiliation of the violator on a public <i>All of Us</i>
+                Research Program webpage.
+              </IndentedListItem>
+              <IndentedListItem>
+                Revoke extant NIH funding and/or prohibit future funding, either permanently or for
+                an explicit period of time
+              </IndentedListItem>
+              <IndentedListItem>
+                Prosecute the violator for breach of a contract with the federal government
+              </IndentedListItem>
+            </IndentedUnorderedList>
+          </li>
+        </IndentedUnorderedList>
+      </label>
+    </ModalBody>
+    <ModalFooter>
+      <Button type='primary' onClick={props.onClose}>Close</Button>
+    </ModalFooter>
+  </Modal>;
+};
+
+export function getDataUseAgreementWidget(submitting, name, initialWork,
+  initialName, initialSanctions, showSanctionModal, errors) {
+  return <FlexColumn style={{
+    borderRadius: '1rem',
+    backgroundColor: colorWithWhiteness(colors.primary, 0.8),
+    padding: '1rem', alignItems: 'flex-start', position: 'relative'
+  }}>
+    {submitting && <SpinnerOverlay/>}
+    <SecondHeader style={{marginTop: 0}}>Agreement:</SecondHeader>
+    <div style={{marginTop: '0.5rem', fontWeight: 600}}>I
+      <DuaTextInput placeholder='FIRST AND LAST NAME' style={{margin: '0 1ex'}}
+                    onChange={(v) => this.setState({name: v})} value={name}
+                    data-test-id='dua-name-input'/>
+      ("Authorized User") have
```

Comment:
This should say `"Authorized Demonstration User"`

Reply:
bump